### PR TITLE
Flex-Tab CoffeeScript to JavaScript I 

### PR DESCRIPTION
**Pull request by MartinSchoeler**
Converted some files in the flex tab package to javascript

### CODE REVIEW

File: packages/rocketchat-ui-flextab/client/tabs/messageSearch.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+});
+
+
+Template.messageSearch.helpers({
+	tSearchMessages() {
+		return t('Search_Messages');
+	},
+
+	searchResultMessages() {
+		return Template.instance().searchResult.get().messages;
```
Comment (rodrigok) [inline packages/rocketchat-ui-flextab/client/tabs/messageSearch.js]: `Template.instance().searchResult.get()` could be empty

**Comment by MartinSchoeler** [comment]
@rodrigok @marceloschmidt could you guys take another look in my pr?

**Comment by rodrigok** [comment]
@marceloschmidt can you change your review?

File: packages/rocketchat-ui-flextab/client/tabs/membersList.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+			]
+		};
+	},
+
+	showUserInfo() {
+		const webrtc = WebRTC.getInstanceByRoomId(this.rid);
+		let videoActive = undefined;
+		if (webrtc && webrtc.localUrl && webrtc.localUrl.get()) {
+			videoActive = webrtc.localUrl.get();
+		} else if (webrtc && webrtc.remoteItems && webrtc.remoteItems.get().length > 0) {
```
Comment (marceloschmidt) [inline packages/rocketchat-ui-flextab/client/tabs/membersList.js]: what if `webrtc.remoteItems.get()` returns undefined?

**Pull request merged**